### PR TITLE
Snow 79080

### PR DESCRIPTION
**Pull request by smtakeda**

### CODE REVIEW

File: Snowflake.Data.Tests/SFBaseTest.cs [hunk 1]
```diff
@@ -59,7 +59,9 @@ public void SFTestSetup()
             if (testConfigs.TryGetValue(connectionKey, out testConnectionConfig))
             {
                 connectionString = String.Format(connectionStringFmt,
+                    testConnectionConfig.protocol,
```
Comment (howryu) [inline Snowflake.Data.Tests/SFBaseTest.cs]: Can you use https as default protocol if not specified? In the example in https://github.com/snowflakedb/snowflake-connector-net/blob/master/README.md, I did not provide protocol and existing customer who is developing against our production environment may break once they update.

Comment (smtakeda) [reply Snowflake.Data.Tests/SFBaseTest.cs]: yes, already set in the constructor of TestConfig

File: Snowflake.Data.Tests/SFBaseTest.cs [hunk 2]
```diff
@@ -101,6 +106,15 @@ public class TestConfig
...
         [JsonProperty(PropertyName = "SNOWFLAKE_TEST_ROLE", NullValueHandling = NullValueHandling.Ignore)]
         internal string role { get; set; }
+
+        [JsonProperty(PropertyName = "SNOWFLAKE_TEST_PROTOCOL", NullValueHandling = NullValueHandling.Ignore)]
+        internal string protocol { get; set; }
+
+        public TestConfig()
+        {
+            this.protocol = "https";
+            this.port = "443";
```
Comment (smtakeda) [inline Snowflake.Data.Tests/SFBaseTest.cs]: @howryu these are the default values.

**Comment by smtakeda** [comment]
ah, we need to check https://ci.appveyor.com/project/howryu/snowflake-connector-net/history

**Pull request merged**